gramps/gui/widgets/fanchart.py

```diff
@@ -534,166 +620,138 @@ def wrap_truncate_layout(self, layout, font, width_pixels):
         if layout.get_line_count() > 1:
             layout.set_text(layout.get_text(), layout.get_line(0).length)
 
+        #2. we check if height is ok
+        w, h = layout.get_pixel_size()
+        if h > height_pixels:
+            if tryrescale:
+                #try to reduce the height
+                fontsize = max(height_pixels / h * font.get_size() /1.1, font.get_size()/2.0)
+                font.set_size(fontsize)
+                layout.set_text(all_text_backup, len(all_text_backup)) # reducing the height allows for more characters
```

Comment:
Change this to:
`layout.set_text(all_text_backup, len(all_text_backup.encode('utf-8'))) # reducing the height allows for more characters`

as mentioned by @Nick-Hall  in https://github.com/gramps-project/gramps/pull/222#issuecomment-262861239